### DISCUSSION TITLE
How to not include polyfills? Especially await polyfill

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [storybookjs/storybook#18854] [Help] [answered]
I am getting an uncaught promise error and I think it's because of `await` is being polyfilled and the polyfill isn't working as it should. I want to not polyfill await for better debugging and performance. How to do that?

**Comment by shilman** [comment, marked as answer]
We're removing all polyfills in the next release, 7.0. Unfortunately I don't know how to get rid of them in the current release
